python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -9027,3 +9027,71 @@ def test_dataframe_add_suffix():
     expected = pdf.add_suffix("_item")
 
     assert_eq(got, expected)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        np.random.randint(-50, 50, (25, 30)),
+        np.random.random_sample((4, 4)),
```

Comment:
Not sure if I'm a fan of randomized input anymore... At least we should use const seed to make them reproducible.